packages/form-validations/src/lang.ts

```diff
@@ -4,5 +4,16 @@ export const lang: Languages = {
   en: {
     required: 'This field is required',
     default: 'Please enter a correct value',
+    email: 'This field requires a valid e-mail address',
+    number: 'This field requires a number',
+    integer: 'This field requires an integer value',
+    url: 'This field requires a valid website URL',
+    tel: 'This field requires a valid telephone number',
+    maxlength: `This fields length must be < ${1}`,
+    minlength: `This fields length must be > ${1}`,
+    min: `Minimum value for this field is ${1}`,
```

Comment:
The replace doesn't work here. Please fix it

Reply:
Fixed.